### DISCUSSION TITLE
How does the --rm or --drop flag work

### DISCUSSION THREAD

**Discussion by samene** [Altinity/clickhouse-backup#662] [General]
We have a backup configured to skip a few tables

```yaml
skip_tables:
  - npo.REF*
  - npo.INSERT_*
```

Then when we do a restore of this backup using `--rm` and `--schema` followed by `--data` we see that the existing tables REF* and INSERT_* are missing and all others are restored.

Is this expected behaviour?

**Comment by Slach** [comment]
`skip_tables`
applied in multiple commands
- `create`, will skip tables during backup creation
- `tables`, will skip tables during list exists tables from clickhouse-server
- `upload`, will skip tables for uploading even if it presents in backup
- `donwload`, will skip tables for downloading even if it presents in backup
- `restore` will skip tables to restore even if it presents in backup

`--rm` or `--drop`  will apply `DROP DATABASE IF EXISTS db` for every databases present in backup
and known nothing about skip_tables pattern

maybe we shall refactoring this behavior to apply DROP TABLE IF EXISTS ... for `--rm` option instead of DROP DATABASE IF EXISTS ...
